renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -970,12 +970,13 @@ bucket when the pixel values are divided between ``minval`` and ``maxval``.
 :param int sample: The multi-sampled sample. Ignored if non-multisampled texture.
 :param int primitive: Debug the pixel from this primitive if there's ambiguity. If set to
   :data:`NoPreference` then a random fragment writing to the given co-ordinate is debugged.
+:param int view: Debug the pixel writing to this view, ignored if set to :data:`NoPreference`.
```

Comment:
It would be good to be explicit here about this referring only to multiview views (since viewports are a different concept and there's room for ambiguity here). Also mentioning that it's ignored if there is no multiview in use.